classes/npm.bbclass

```diff
@@ -211,7 +167,7 @@ python fetch_npm() {
         json_objs = {'dependencies': { npmpn: '' }}
         json.dump(json_objs, outfile, indent=2)
 
-    runcmd(d, "npm ci --global-style --ignore-scripts --verbose", "fetch-tmp")
+    runcmd(d, "npm ci --global-style --ignore-scripts --verbose")
```

Comment:
Why dropping "fetch-tmp"?

Reply:
I suppose that is because we are running this in a schroot, no longer in WORKDIR.